### PR TITLE
More understandable error for scheduler configuration validation

### PR DESCRIPTION
**Pull request by kerthcet**
#### What type of PR is this?

/kind cleanup
/sig scheduling

#### What this PR does / why we need it:

The original error makes me think that I should set the `profiles[1].plugins.queueSort` to all profiles. However, the truth is 
we should set the queueSort same for all profiles.

```
E1219 10:20:48.935102       1 run.go:74] "command failed" err="profiles[1].plugins.queueSort: Invalid value: config.PluginSet{Enabled:[]config.Plugin{config.Plugin{Name:\"PrioritySort\", Weight:0}}, Disabled:[]config.Plugin{config.Plugin{Name:\"*\", Weight:0}}}: has to match for all profiles"
```

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by SataQiu — APPROVED** [review]
LGTM

File: pkg/scheduler/apis/config/validation/validation.go [hunk 1]
```diff
@@ -273,11 +273,11 @@ func validateCommonQueueSort(path *field.Path, profiles []config.KubeSchedulerPr
 			curr = profiles[i].Plugins.QueueSort
 		}
 		if !cmp.Equal(canon, curr) {
-			errs = append(errs, field.Invalid(path.Index(i).Child("plugins", "queueSort"), curr, "has to match for all profiles"))
+			errs = append(errs, field.Invalid(path.Index(i).Child("plugins", "queueSort"), curr, "queueSort must be the same for all profiles"))
 		}
 		for _, cfg := range profiles[i].PluginConfig {
 			if cfg.Name == queueSortName && !cmp.Equal(queueSortArgs, cfg.Args) {
```
Comment (alculquicondor) [inline pkg/scheduler/apis/config/validation/validation.go:279]: Uhm... we should have used `equality.Semantic.DeepEqual` here.

Comment (kerthcet) [reply pkg/scheduler/apis/config/validation/validation.go:279]: Make sense for api validation, append a new fix.

**Pull request merged**